mscore/inspector/inspector.cpp

```diff
@@ -399,8 +403,13 @@ void Inspector::update(Score* s)
                               break;
                         }
                   }
+            Q_ASSERT(ie);
```

Comment:
this indeed makes MuseScore crash on selecting any layoutbreak but section breaks, when it is a Debug build.
Simply doing
```
      if (ie)
            return;
```
should fix this